### ISSUE TITLE
Forums are down

### ISSUE THREAD

**Issue by realies** [ggsava/block-this#26]
https://forum.block-this.com/ returns `502 Bad Gateway`

**Comment by ggsava** [comment]
Yes I'm aware. Unfortunately the forum has been under attack by Chinese bots for a while.. I'm looking for ways to prevent it.

Thanks a lot for reporting it anyway.

**Comment by hxalfa** [comment]
Try to use Cloudflare free plan. It is good in preventing bots and DDoS attacks.
